sdk/keyvault/azure-keyvault-certificates/migration_guide.md

```diff
@@ -0,0 +1,210 @@
+certificate = create_certificate_poller.result()
+```
+
+If you would like to check the status of your certificate creation, you can call `status()` on the poller or `get_certificate_operation` with the name of the certificate.
+
+### Retrieve a certificate
+
+In `azure-keyvault` you could retrieve a certificate (as a `CertificateBundle`) by using `get_certificate` and specifying the desired vault endpoint, certificate name, and certificate version. You could retrieve the versions of a certificate with the `get_certificate_versions` method, which returned an iterator-like object.
+
+```python
+certificate_versions = client.get_certificate_versions(
+    vault_base_url="https://my-key-vault.vault.azure.net/",
+    certificate_name="cert-name"
+)
+
+for certificate_version in certificate_versions:
+    certificate = client.get_certificate(
+        vault_base_url="https://my-key-vault.vault.azure.net/",
+        certificate_name="cert-name",
+        certificate_version=certificate_version
+    )
+```
+
+Now in `azure-keyvault-certificates` you can retrieve the latest version of a certificate (as a `KeyVaultCertificate`) by using `get_certificate` and providing a certificate name.
+
+```python
+certificate = certificate_client.get_certificate(certificate_name="cert-name")
+
+print(certificate.name)
+print(certificate.properties.version)
+print(certificate.policy.issuer_name)
+```
+
+You can use `get_certificate_version` to retrieve a specific version of a certificate.
+
+```python
+certificate = certificate_client.get_certificate_version(certificate_name="cert-name", version="cert-version")
+```
+
+### List properties of certificates
+
+In `azure-keyvault` you could list the properties of certificates in a specified vault with the `get_certificates` method. This returned an iterator-like object containing `CertificateItem` instances.
+
+```python
+certificates = client.get_certificates(vault_base_url="https://my-key-vault.vault.azure.net/")
+
+for certificate in certificates:
+    print(certificate.attributes.created)
+    print(certificate.x509_thumbprint)
+```
+
+Now in `azure-keyvault-certificates` you can list the properties of certificates in a vault with the `list_properties_of_certificates` method. This returns an iterator-like object containing `CertificateProperties` instances.
+
+```python
+certificates = certificate_client.list_properties_of_certificates()
+
+for certificate in certificates:
+    print(certificate.name)
+    print(certificate.x509_thumbprint)
+```
+
+### Delete a certificate
+
+In `azure-keyvault` you could delete all versions of a certificate with the `delete_certificate` method. This returned information about the deleted certificate (as a `DeletedCertificateBundle`), but you you could not poll the deletion operation to know when it completed. This would be valuable information if you intended to permanently delete the deleted certificate with `purge_deleted_certificate`.
```

Comment:
```suggestion
In `azure-keyvault` you could delete all versions of a certificate with the `delete_certificate` method. This returned information about the deleted certificate (as a `DeletedCertificateBundle`), but you could not poll the deletion operation to know when it completed. This would be valuable information if you intended to permanently delete the deleted certificate with `purge_deleted_certificate`.
```